osf/models/preprint_provider.py

```diff
@@ -14,22 +15,22 @@
 
 class PreprintProvider(ObjectIDMixin, BaseModel):
     name = models.CharField(null=False, max_length=128)  # max length on prod: 22
-    logo_name = models.CharField(null=True, blank=True, max_length=128)  # max length on prod: 17
-    header_text = models.TextField(default='', blank=True)
-    description = models.CharField(null=True, blank=True, max_length=256)  # max length on prod: 56
-    banner_name = models.CharField(null=True, blank=True, max_length=128)  # max length on prod: 19
+    description = models.TextField(default='', blank=True)
     domain = models.URLField(blank=True, default='', max_length=200)
     domain_redirect_enabled = models.BooleanField(default=False)
     external_url = models.URLField(null=True, blank=True, max_length=200)  # max length on prod: 25
     email_contact = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 23
     email_support = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 23
     example = models.CharField(null=True, blank=True, max_length=20)  # max length on prod: 5
     access_token = EncryptedTextField(null=True, blank=True)
-    advisory_board = models.TextField(null=True, blank=True)
+    advisory_board = models.TextField(default='', blank=True)
     social_twitter = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
     social_facebook = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
     social_instagram = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
+    footer_links = models.TextField(default='', blank=True)
     share_source = models.CharField(blank=True, max_length=200)
+    allow_submissions = models.BooleanField(default=True)
+    additional_providers = fields.ArrayField(models.CharField(max_length=200), default=list, blank=True)
```

Comment:
Yep, I went and found the same thing because I'd never seen that before, or at least I didn't remember. You should checkout [dash](https://kapeli.com/dash). Then you can get them when the power is out and the internet doesn't work. :high-five: